roles/nat64_appliance/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure nat64-appliance working directory
+  ansible.builtin.file:
+    path: "{{ cifmw_nat64_appliance_dir }}"
+    state: directory
+    mode: '0755'
+
+- name: Make sure python3-pip is installed
+  become: true
+  ansible.builtin.package:
+    name:
+      - python3-pip
+      - qemu-img
+    state: present
+
+- name: Install diskimage-builder in virtualenv
+  ansible.builtin.pip:
+    virtualenv_command: "python3 -m venv"
+    virtualenv: "{{ cifmw_nat64_appliance_dir }}/venv"
```

Comment:
Would it be possible to move that venv in another location? Else it would end in the gathered logs, leading to a huge amount of useless files.
You may want to use `cifmw_nat64_appliance_basedir/venv` (and build the cifmw_nat64_appliance_dir by appending the artifacts/roles/.... to that basedir).

Reply:
Done